### PR TITLE
Update sync-endpoint-manual-setup.rst

### PR DESCRIPTION
**Pull request by icytailovo**
addresses odk-x/tool-suite-X#207

#### What is included in this PR?
Update some instructions to make it easier to read and go over

#### What new issues will need to be opened because of this PR?
For the 3rd instruction on the custom database section, hope to have detailed instruction on setting up jdbc.properties

#### What is left to be done in the addressed issue?
For the 3rd instruction on the custom database section, hope to have detailed instruction on setting up jdbc.properties

### CODE REVIEW

File: odkx-src/sync-endpoint-manual-setup.rst [hunk 1]
```diff
@@ -92,9 +94,9 @@ Setup instructions:
 
     $ docker build --pull -t odk/phpldapadmin phpldapadmin
 
-  10. Enter your hostname in the :code:`security.server.hostname` field in the :file:`security.properties` file (under the directory :file:`config/sync-endpoint`). You can also choose to enable :ref:`Anonymous access<sync-anonymous>` on your ODK-X Sync Endpoint by configuring the same :file:`security.properties` file.
+  10. Enter your hostname in the :code:`security.server.hostname` field (if such field doesn't exists, create one at the bottom of file) in the :file:`security.properties` file (under the directory :file:`config/sync-endpoint`). You can also choose to enable :ref:`Anonymous access<sync-anonymous>` on your ODK-X Sync Endpoint by configuring the same :file:`security.properties` file.
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-manual-setup.rst:97]: "if such field doesn't exists" should be "if this field does not exist"

File: odkx-src/sync-endpoint-manual-setup.rst [hunk 2]
```diff
@@ -92,9 +94,9 @@ Setup instructions:
...
 
-  11. If you're not using the standard ports (80 for *HTTP* and 443 for *HTTPS*) enter the ports you're using in the :code:`security.server.port` and :code:`security.server.securePort` fields in the :file:`security.properties`. Then edit the **ports** section under the **sync** section in :file:`docker-compose.yml` to be :code:`YOUR_PORT:8080`.
+  11. If you're not using the standard ports (80 for *HTTP* and 443 for *HTTPS*) enter the ports you're using in the :code:`security.server.port` and :code:`security.server.securePort` fields in the :file:`security.properties` (if such fields doesn't exists, create them at the bottom of file). Then add the **ports** section under the **sync** section in :file:`docker-compose.yml` to be :code:`YOUR_PORT:8080`.
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-manual-setup.rst:99]: same as previous, "create it at the bottom of file"
Also the ports section might sometimes exist, maybe "add or edit"

**Review by elmps2018 — COMMENTED** [review]
Thanks for working on these clarifications!

File: odkx-src/sync-endpoint-manual-setup.rst [hunk 3]
```diff
@@ -96,7 +96,7 @@ Setup instructions:
...
 
-  11. If you're not using the standard ports (80 for *HTTP* and 443 for *HTTPS*) enter the ports you're using in the :code:`security.server.port` and :code:`security.server.securePort` fields in the :file:`security.properties` (if such fields doesn't exists, create them at the bottom of file). Then add the **ports** section under the **sync** section in :file:`docker-compose.yml` to be :code:`YOUR_PORT:8080`. 
+  11. If you're not using the standard ports (80 for *HTTP* and 443 for *HTTPS*) enter the ports you're using in the :code:`security.server.port` and :code:`security.server.securePort` fields in the :file:`security.properties` (if such field doesn't exists, create it at the bottom of file). Then add or edit the **ports** section under the **sync** section in :file:`docker-compose.yml` to be :code:`YOUR_PORT:8080`.
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-manual-setup.rst:99]: Should be "if such a field does not exist, create it"

File: odkx-src/sync-endpoint-manual-setup.rst [hunk 4]
```diff
@@ -124,9 +124,41 @@ Setup instructions:
     .. code-block:: console
 
        $ docker stack deploy -c docker-compose.yml -c docker-compose-https.yml syncldap
+
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-manual-setup.rst:127]: should be "during the docker stack deploy process"

**Review by elmps2018 — COMMENTED** [review]
Thanks for working on these!

**Comment by elmps2018** [comment]
Thanks for working on these edits! @wbrunette or @linl33 can you review the conent?

**Comment by wbrunette** [comment]
@icytailovo you stated "For the 3rd instruction on the custom database section, hope to have detailed instruction on setting up jdbc.properties" do you plan to add that? Otherwise looks good

**Comment by icytailovo** [comment]
> @icytailovo you stated "For the 3rd instruction on the custom database section, hope to have detailed instruction on setting up jdbc.properties" do you plan to add that? Otherwise looks good

I will work on that.

**Comment by icytailovo** [comment]
@wbrunette I am still working on it and I will try to finish it recently.

**Pull request merged**